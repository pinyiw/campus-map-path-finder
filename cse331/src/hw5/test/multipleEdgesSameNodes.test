# Create a graph
CreateGraph g

# Add nodes
AddNode g a
AddNode g b
AddNode g c

# Add edges
AddEdge g a b e1
AddEdge g a b e2
AddEdge g a b e3
AddEdge g b c e4
AddEdge g b c e5

# Print the outgoing edges from a and b
ListChildren g a
ListChildren g b