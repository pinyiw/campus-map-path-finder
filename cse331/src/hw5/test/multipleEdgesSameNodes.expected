# Create a graph
created graph g

# Add nodes
added node a to g
added node b to g
added node c to g

# Add edges
added edge e1 from a to b in g
added edge e2 from a to b in g
added edge e3 from a to b in g
added edge e4 from b to c in g
added edge e5 from b to c in g

# Print the outgoing edges from a and b
the children of a in g are: b(e1) b(e2) b(e3)
the children of b in g are: c(e4) c(e5)